src/lossless.rs

```diff
@@ -673,11 +674,9 @@ impl ColorCache {
         self.color_cache[index as usize] = color;
     }
 
-    fn lookup(&self, index: usize) -> Result<[u8; 4], DecodingError> {
-        match self.color_cache.get(index) {
-            Some(&value) => Ok(value),
-            None => Err(DecodingError::BitStreamError),
```

Comment:
This branch should actually be unreachable because only valid indexes can be represented in the bitstream, so I've converted it to a panic